apps/us.zoom.Zoom/eos-get-us.zoom.Zoom.desktop.in

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Version=1.0
+Name=Get Zoom
+Exec=/usr/bin/eos-install-app-helper --app-id us.zoom.Zoom --remote flathub --branch stable --required-archs x86_64 %U
+Terminal=false
+Icon=/var/lib/flatpak/appstream/flathub/x86_64/active/icons/128x128/us.zoom.Zoom.png
```

Comment:
What will happen if we dont have this cached file present (e.g. on new installs) or is it guaranteed to always exist?

Reply:
Great question. The answer is that you get a blank space where the icon should be.

![Capture d’écran de 2020-05-11 21-37-24](https://user-images.githubusercontent.com/86760/81609340-a260fd00-93cf-11ea-8c3e-de1a9b0ec219.png)
![Capture d’écran de 2020-05-11 21-37-40](https://user-images.githubusercontent.com/86760/81609360-a8ef7480-93cf-11ea-9f36-2614398b0f12.png)

However, I did check, and the icons are correctly cached in this location on fresh images.

It's not *guaranteed* to exist, as such. However, Flathub requires that applications provide a 64×64 and 128×128 icon and that they match the app ID. So, if it goes away, it means something like, "Zoom has been renamed" or "Zoom has been removed", in which case the launcher will stop working anyway.

Seem OK?

Reply:
Yep, seems ok to me.